.codeclimate.yml

```diff
@@ -0,0 +1,6 @@
+languages:
+  JavaScript: true
+
+exclude_paths:
+- "database_reports/*"
+- "migrations/*"
```

Comment:
Seems strange to me that the example file would include a test directory as excluded, but you're right that it's not showing up in those results.